assets/multiqc_config.yaml

```diff
@@ -16,6 +16,8 @@ run_modules:
     - snpeff
     - quast
     - cutadapt
+    #- ivar
```

Comment:
Are these ivar and varscan2 sections all meant to be commented out?

Reply:
Yep 👍 Those modules havent been added to MultiQC yet :) Will be there in the next release (soon).